app/components/SingleProduct.js

```diff
@@ -73,6 +73,7 @@ const mapState = (state) => {
     selectedProduct: state.selectedProduct,
     cart: state.cart,
     user: state.user,
+    review: state.review
```

Comment:
this isn't going to work because we aren't storing the review separately on the state. You can just leave this line out and we can access the view via the selectedProduct (as it will eagerly load its associated reviews)